src/components/Table/BaseTable.tsx

```diff
@@ -45,6 +48,7 @@ declare module "@tanstack/table-core" {
 }
 
 type SpruceTableProps = {
+  className?: string;
```

Comment:
Would `className` maybe captured by [this declaration](https://github.com/mongodb/leafygreen-ui/blob/9a35ec367bb88f471082d58c535937e2b2be9d7f/packages/table/src/Table/Table.types.ts#L7) in `TableProps` and subsequently `args` below? If removing this errors though then LGTM!

Reply:
good catch!